core/modules/replica/testNamedMutexRegistry.cc

```diff
@@ -171,6 +172,18 @@ BOOST_AUTO_TEST_CASE(NamedMutexRegistryTest2) {
 
 BOOST_AUTO_TEST_CASE(NamedMutexRegistryTest3) {
 
+    bool testMaxCpuUsage = false;
+    for (int i = 1; i < framework::master_test_suite().argc; ++i) {
```

Comment:
please somehow document the use of this command line argument. (IMO it doesn't have to be formal but it should be easy to understand.)